content/javascript/general-authoring/index.md

```diff
@@ -171,47 +171,48 @@ Use single quotes `''` for strings.
 
 ```javascript
 // bad
-var name = "Bob Parr";
+const name = "Bob Parr";
 
-// good
-var name = 'Bob Parr';
+const fullName = "Bob " + this.lastName;
 
-// bad
-var fullName = "Bob " + this.lastName;
+var errorMessage = 'This is a super long error that was thrown because ' +
```

Comment:
Is the goal of this PR "kill all `var` s"? If so, this one escaped.

Reply:
It wasn't, that was another PR but in any case I'll remove it. :)